proto/offers.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+package offers;
+
+service Offers {
+    rpc PayOffer (PayOfferRequest) returns (PayOfferResponse);
+}
+
+message PayOfferRequest {
+   string offer = 1;
+   optional uint64 amount = 2;
+}
+
+message PayOfferResponse {
+    string payment_preimage = 2;
+}
```

Comment:
Can we include more details about the payment in this response?

Reply:
Sure, I was just using the minimum here as a start, but happy to add whatever info you'd find useful.